### DISCUSSION TITLE
سندباکس

### DISCUSSION THREAD

**Discussion by sajjadarashhh** [Sina-Soltani/Parbad#228] [Q&A]
توی زرین پال یه مد سند باکس هم داریم که خیلی نیاز هست برای پروژه ها اونو چطوری هندل میکنیین!؟
دارینش!؟

**Comment by Sina-Soltani** [comment]
https://github.com/Sina-Soltani/Parbad/blob/master/src/Parbad.Gateway/ZarinPal/src/ZarinPalGatewayAccount.cs#L15
